content/posts/2023/previewing-workspace-files.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Previewing workspace files"
+date: 2023-11-13T14:00:00+01:00
+---
+
+For a long time, the REANA web UI has allowed users to preview certain types of files present in the
+workflow workspace. This has proven to be very convenient for users who want to quickly glance through
+images and text files in their workspaces without having to download them. 
+Expanding on this, recent enhancements in REANA now include the previewing of PDF and ROOT files, 
+increasing the number of file types that can be inspected within the browser.
+
+<!--more-->
+
+## Images and text files preview
+
+Probably amongst the most common file types in a workflow workspace are images and text files. The web UI has
+always allowed to preview them from a modal window opened just by clicking on the name of the file in the workspace list.
+From there, you can also download the file if needed. Note that text files are not limited to `.txt` files, 
+but they also include logs, code, and other text-based files.
+
+![Text file preview](/images/text-file-preview.png)
+
+## HTML files preview
+
+Since REANA 0.8.1, users have been able to preview HTML files directly in the web UI.
+This feature remains a valuable asset, allowing for example immediate visibility into Snakemake
+reports and other HTML-based documentation pages directly within the browser.
+
+![HTML file preview](/images/snakemake-support-report.png)
```

Comment:
BTW it may be nicer  to use the CMS H4l example which produces a richer report:

![report-h4l](https://github.com/reanahub/blog.reana.io/assets/517546/f408dbf5-4a49-4b77-a93e-e3196dabd388)

Well, speaking of this, the Snakemake reports can be much richer, for example see [this upstream example](https://slides.com/johanneskoester/snakemake-short#/18):

![report-rich](https://github.com/reanahub/blog.reana.io/assets/517546/5b9e146f-8366-4e2b-9d2e-49504826685b)

However we don't have such a rich report defined yet in any of our REANA demo examples.

It may be perhaps worth to take the time and enrich either the RooFit example or the CMS H4l example to produce such a richer Snakemake report including final plots with descriptions etc?